### DISCUSSION TITLE
Option to disable bed missing / obstructed message when using respawn settings.

### DISCUSSION THREAD

**Discussion by mibby** [EssentialsX/Essentials#3983] [Ideas and Feature Suggestions] [answered]
RE: https://github.com/EssentialsX/Essentials/issues/3836#issuecomment-770402204

Is there any way to disable this message from appearing when you die and _don't_ have a home bed or respawn anchor? I just want to silently teleport players to spawn if those aren't present. The message only occurs when special respawn condition settings are enabled. When you _don't_ use anchors / bed home and just force players back to spawn, there is no message on death.

![link2](https://i.imgur.com/p1GfwWR.png)

```
# What priority should we use for handling respawns?
# Set this to none, if you want vanilla respawning behaviour.
# Set this to lowest, if you want Multiverse to handle the respawning.
# Set this to high, if you want EssentialsSpawn to handle the respawning.
# Set this to highest, if you want to force EssentialsSpawn to handle the respawning.
respawn-listener-priority: high

# What priority should we use for handling spawning on joining the server?
# See respawn-listener-priority for possible values.
# Note: changing this may impact or break spawn-on-join functionality.
spawn-join-listener-priority: high

# When users die, should they respawn at their first home or bed, instead of the spawnpoint?
respawn-at-home: false

# When users die, should EssentialsSpawn respect users' respawn anchors?
respawn-at-anchor: true

respawn-at-home-bed: true
```

**Comment by mdcfe** [comment, marked as answer]
The white message is a vanilla message introduced in Minecraft 1.15.x (`block.minecraft.spawn.not_valid`). Spigot's `PlayerRespawnEvent` [does not have a method to suppress this](https://hub.spigotmc.org/javadocs/spigot/org/bukkit/event/player/PlayerRespawnEvent.html), so there's nothing we can do in EssentialsX about this.

**Reply by mibby** [reply]
Is there no way to prevent it from being triggered and getting the player's Bed/Anchor point manually for respawn handling? The message doesn't happen when you just use EssentialsXSpawn to teleport to the main spawn point instead of beds/homes/anchors.

**Reply by mdcfe** [reply]
The message isn't sent as a result of anything EssentialsX Spawn does. As I understand it, the server sends this message when checking bed locations during respawns, and this is totally outside of plugins' control, so changing Spawn's handling of respawns is unlikely to stop this.
